src/components/Footer.jsx

```diff
@@ -57,6 +57,11 @@ const Footer = () => (
         />
       </div>
     </div>
+    <div className="mx-auto font-poppins font-normal text-dimWhite text-[20px] leading-[30.8px] max-w-[700px] my-1 ">
```

Comment:
I'd prefer something like 12 px on tablets and larger devices.
An even smaller size on phones so that it fits in one line.
Make sure that the text remains centered on all screen sizes